src/mon/MonCommands.h

```diff
@@ -330,7 +330,7 @@ COMMAND("mds set_max_mds " \
 	"set max MDS index", "mds", "rw", "cli,rest")
 COMMAND("mds set " \
 	"name=var,type=CephChoices,strings=max_mds|max_file_size"
-	"|allow_new_snaps|inline_data|allow_multimds|allow_dirfrags " \
+	"|allow_new_snaps|inline_data|allow_multimds|allow_dirfrags|balancer " \
```

Comment:
Please add this to "fs set" instead of "mds set" -- mds set is the legacy mode that operates only on the default filesystem, fs set is the new hotness which lets you work on any filesystem